src/pages/landing/ebooks/components/_header.js

```diff
@@ -0,0 +1,222 @@
+        padding: 40px 15px;
+    }
+`
+
+const TopHeaderImgWrapper = styled(Flex)`
+    justify-content: flex-start;
+    margin: 0;
+    padding: 0;
+    position: relative;
+
+    @media ${device.tablet} {
+        order: 2;
+    }
+
+    @media screen and (max-width: 991px) {
+        margin-top: 110px;
+    }
+
+    @media screen and (max-width: 768px) {
+        margin-top: -70px;
+    }
+`
+const HeaderImage = styled(QueryImage)`
+    width: ${(props) => (props.imgWidth ? `${props.imgWidth}px` : '557px')};
+    height: ${(props) => (props.imgHeight ? `${props.imgHeight}px` : '703px')};
+    position: relative;
+    top: 75px;
+    margin: 0;
+
+    @media ${device.laptop} {
+        margin: 0 auto;
+        width: 100%;
+        height: 100%;
+    }
+
+    @media ${device.tablet} {
+        top: 110px;
+        height: auto;
+        margin: initial;
+        width: 80%;
+        margin-left: ${(props) => (props.imgWidth < 600 ? '-50px' : '-15px')};
+    }
+
+    @media ${device.tabletS} {
+        width: 100%;
+        margin-left: ${(props) => (props.imgWidth < 600 ? '-70px' : '-20px')};
+    }
+`
+
+const ContentWrapper = styled.div`
+    width: 100%;
+    float: right;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+`
+
+const Content = styled.div`
+    width: 100%;
+    margin: 0;
+    padding: 0 30px 0 45px;
+
+    @media ${device.laptopL} {
+        padding-left: 45px;
+        margin-right: 30px;
+    }
+
+    @media ${device.laptop} {
+        padding: 0 60px 0 25px;
+        max-width: 600px;
+    }
+
+    @media ${device.tablet} {
+        float: none;
+        max-width: 360px;
+        padding: 0;
+        margin: 0 auto;
+        justify-content: center;
+    }
+
+    .title {
+        @media screen and (min-width: 1200px) {
+            max-width: 600px;
+        }
+        @media ${device.laptopL} {
+            max-width: 400px;
+        }
+        @media ${device.tablet} {
+            max-width: 300px;
+        }
+    }
+`
+
+const AuthorText = styled(Text)`
+    font-style: italic;
+`
+
+const AuthorNameText = styled(Text).attrs({ as: 'span' })``
+
+const SignupWrapper = styled.div`
+    margin-bottom: 10px;
+`
+
+const HeaderSection = ({
+    mainHeaderImage,
+    imgWidth,
+    imgHeight,
+    introSub,
+    introMain,
+    authorDesc,
+    authorName,
+    bg,
+    bgMobile,
+    ebook_utm_code,
+}) => {
+    return (
+        <MainWrapper bg={bg} bgMobile={bgMobile}>
+            <HeaderBody>
+                <TopHeaderImgWrapper>
+                    <HeaderImage
+                        data={mainHeaderImage}
+                        alt="ebook"
+                        imgWidth={imgWidth}
+                        imgHeight={imgHeight}
+                        height="100%"
+                    />
+                </TopHeaderImgWrapper>
+                <ContentWrapper>
+                    <Content>
+                        <Header
+                            className="sub-title"
```

Comment:
Styled components handles class names for us. You can extend the `Header` component if you require more specific styles for different breakpoints. Same for the header below.